src/components/AgreementLineSections/FormCoverage/FormCoverage.test.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import '@folio/stripes-erm-components/test/jest/__mock__';
+import { TestForm, renderWithIntl } from '@folio/stripes-erm-components/test/jest/helpers';
+import { Accordion, KeyValue } from '@folio/stripes-testing';
+import FormCoverage from './FormCoverage';
+import { line, values, resource, handlers, initialValues } from './testResources';
+import translationsProperties from '../../../../test/helpers';
+
+jest.mock('../../CoverageFieldArray', () => () => <div>CoverageFieldArray</div>);
+jest.mock('@folio/stripes-erm-components', () => ({
+  ...jest.requireActual('@folio/stripes-erm-components'),
+  SerialCoverage: () => <div>SerialCoverage</div>,
+}));
+
+const onSubmit = jest.fn();
+
+describe('FormCoverage', () => {
+  let renderComponent;
+  describe('with no initialValues', () => {
+    beforeEach(() => {
+      renderComponent = renderWithIntl(
+        <TestForm onSubmit={onSubmit}>
+          <FormCoverage
+            line={line}
+            resource={resource}
+            values={values}
+          />
+        </TestForm>,
+        translationsProperties
+      );
+    });
+
+    test('renders the Coverage accordion', async () => {
+      await Accordion('Coverage').exists();
+    });
+
+    test('renders the expected value', async () => {
+      await KeyValue('Default coverage').exists();
```

Comment:
The key here I believe is default coverage, but we are not checking if the value exists or not.

example usage: [here]( https://github.com/folio-org/ui-agreements/blob/0c06fd070856c2d80eb3bc4962acfbebf6cce8ee/src/components/AgreementLineSections/Info/Info.test.js#L36)